crates/spirv-std/src/lib.rs

```diff
@@ -39,8 +39,6 @@
 mod math_ext;
 pub use math_ext::MathExt;
 
-pub use glam;
```

Comment:
@XAMPPRocky explicitly added this in https://github.com/EmbarkStudios/rust-gpu/pull/220, so we should figure out if we want this or not (I said :+1: to her adding it, so I think we should keep it).

Also, there's some issues with overriding versions in cargo.toml, in particular people referencing crates through a git reference will not pick up the workspace override, so keeping the override in the crate instead of the workspace is good.